### PR TITLE
Feature/rds phase 2

### PR DESCRIPTION
**Pull request by Jkovarik**
**Summary:** Summary of changes

Addresses [CUMULUS-1](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1)

This is a task/utility PR to merge RDS Phase 2 to main. 

This is a merge directly after 

## Changes

* Phase 2 feature branch merge to master, PR direct after:  https://ci.earthdata.nasa.gov/browse/CUM-CBA3070-1

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

File: bamboo/set-bamboo-env-variables.sh [hunk 1]
```diff
@@ -144,7 +144,7 @@ fi
 # Target master by default.
 # Update with appropriate conditional
 # when creating a feature branch.
-export PR_BRANCH=master
+export PR_BRANCH='feature/rds-phase-2'
```
Comment (botanical) [inline bamboo/set-bamboo-env-variables.sh:147]: Should we change this back to `master`? 🤔

File: example/lambdas/ftpPopulateTestLambda/package.json [hunk 2]
```diff
@@ -19,21 +19,11 @@
     "access": "private"
```
Comment (botanical) [inline example/lambdas/ftpPopulateTestLambda/package.json:19]: This isn't really relevant to this PR but I am seeing this locally in my editor
```
Value is not accepted. Valid values: "public", "restricted"
```

File: packages/api/ecs/async-operation/package.json [hunk 3]
```diff
@@ -20,9 +20,10 @@
     "test": "../../../../node_modules/.bin/ava"
   },
   "dependencies": {
-    "@cumulus/aws-client": "10.1.1",
-    "@cumulus/db": "10.1.1",
-    "@cumulus/logger": "10.1.1",
+    "@cumulus/aws-client": "10.1.0",
+    "@cumulus/db": "10.1.0",
+    "@cumulus/es-client": "10.1.2-alpha.0",
+    "@cumulus/logger": "10.1.0",
```
Comment (botanical) [inline packages/api/ecs/async-operation/package.json:26]:
```suggestion
    "@cumulus/aws-client": "10.1.1",
    "@cumulus/db": "10.1.1",
    "@cumulus/es-client": "10.1.2-alpha.0",
    "@cumulus/logger": "10.1.1",
```

File: bamboo/set-bamboo-env-variables.sh (hunk 1 above)
Comment (Jkovarik) [reply bamboo/set-bamboo-env-variables.sh:147]: Good catch, yes.   I'll set a PR to update it.

File: example/lambdas/ftpPopulateTestLambda/package.json (hunk 2 above)
Comment (Jkovarik) [reply example/lambdas/ftpPopulateTestLambda/package.json:19]: Yeah, that's not great.   We should update this on the main branch.

File: packages/api/ecs/async-operation/package.json (hunk 3 above)
Comment (Jkovarik) [reply packages/api/ecs/async-operation/package.json:26]: Hmm, yes those got overlooked in the 10.1.1 release.     Updating.

Comment (Jkovarik) [reply packages/api/ecs/async-operation/package.json:26]: The branch is protected, so updates here: https://github.com/nasa/cumulus/pull/2896

File: bamboo/set-bamboo-env-variables.sh (hunk 1 above)
Comment (Jkovarik) [reply bamboo/set-bamboo-env-variables.sh:147]: https://github.com/nasa/cumulus/pull/2896

File: packages/api/ecs/async-operation/package.json (hunk 3 above)
Comment (Jkovarik) [reply packages/api/ecs/async-operation/package.json:26]: More specifically a4429e206a5d04bfdc45e1e4520cf1a7ca9a184f

Comment (botanical) [reply packages/api/ecs/async-operation/package.json:26]: Hmm I don't see the updates in that PR

**Comment by Jkovarik** [comment]
With the merge of #2896 and #2898 I believe all commentary/CI failures from this review have been addressed.

**Pull request merged**